src/components/SpeechRec.vue

```diff
@@ -115,7 +115,9 @@ export default {
     resultHandler(event) {
       this.$emit('transcription', event.results[event.results.length - 1][0].transcript);      
       if (event.results[event.results.length - 1].isFinal) {
-        this.$emit('transcriptionComplete', event.results);
+        //Transcript as String
+        const transcriptString = event.results[event.results.length - 1][0].transcript;
```

Comment:
this `transcriptString` is the same as the value passed to the transcription event - if you declare it before that one you can reuse the same object - it will be clearer that they both get the same data then

Reply:
updated here https://github.com/artificialsolutions/teneo-web-chat/pull/155/commits/5f388b767f8e3f060dba39a24c354ef0a4356598